pyTigerGraph/gds/gsql/dataloaders/edge_loader.gsql

```diff
@@ -67,12 +67,23 @@ CREATE QUERY edge_loader_{QUERYSUFFIX}(
         IF kafka_address IS NOT NULL THEN
             # Write to kafka
             IF security_protocol IS NOT NULL THEN
-                IF write_to_kafka(kafka_address, kafka_topic, "edge_batch_" + int_to_string(batch_id), @@e_batch, security_protocol, sasl_mechanism, sasl_username, sasl_password, ssl_ca_location)!=0 THEN 
-                    @@kafka_error += ("Failed sending edge batch " + int_to_string(batch_id) + "\n");
+                temp = write_to_kafka(kafka_address, kafka_topic, "edge_batch_" + int_to_string(batch_id), @@e_batch, security_protocol, sasl_mechanism, sasl_username, sasl_password, ssl_ca_location);
```

Comment:
Is the `temp` var declared? GSQL is strongly typed and it might not compile becuase temp is not defined. Similarly for all the temp_* vars in other files.